src/PKSim.Core/ICoreUserSettings.cs

```diff
@@ -57,5 +57,10 @@ public interface ICoreUserSettings : OSPSuite.Core.ICoreUserSettings
       ///    Path of the template database containing the template building block
       /// </summary>
       string TemplateDatabasePath { get; set; }
+
+      /// <summary>
+      ///    Default grouping mode for grouping presenter
+      /// </summary>
+      ParameterGroupingModeId DefaultParameterGroupingMode { get; set; }
```

Comment:
Why is this movd here? What does it have to do with Core Settings? This is purely presentational. Explain

Reply:
It was in IUserSettings, but it was moved so that IUserSettings didn't need to be registered. A better spot for it is probably IPresentationUserSettings, which is registered in the starter. 

This property is required by ParameterGroupsPresenter.

Reply:
yep so it should be moved to IPresentationUserSettings